native/src/modules/endpoint/hocs/withPayloadProvider.js

```diff
@@ -95,33 +95,28 @@ const withPayloadProvider = <S: { dispatch: Dispatch<StoreActionType> }, R: {}>
       if (props.status === 'routeNotInitialized') {
         return null
       } else if (props.status === 'error') {
-        return <ScrollView refreshControl={<RefreshControl onRefresh={refreshIfPossible} refreshing={false} />}
-                           contentContainerStyle={{ flexGrow: 1 }}>
+        return <ScrollView contentContainerStyle={{ flexGrow: 1 }}>
           <FailureContainer tryAgain={refreshIfPossible} message={props.message} code={props.code} />
         </ScrollView>
       } else if (props.status === 'languageNotAvailable') {
         return <LanguageNotAvailableContainer languages={props.availableLanguages}
                                               changeLanguage={changeUnavailableLanguage} />
       } else if (props.status === 'loading') {
-        console.log('render loading', props.progress)
         return timeoutExpired
-          ? <ScrollView refreshControl={<RefreshControl refreshing />} contentContainerStyle={{ flexGrow: 1 }}
-                      keyboardShouldPersistTaps='always'>
+          ? <ScrollView contentContainerStyle={{ flexGrow: 1 }} keyboardShouldPersistTaps='always'>
```

Comment:
This should include a RefreshControl. `refreshing` if no `ProgressContainer` is rendered, i.e. if `props.innerProps && props.dispatch` is truthy.

Reply:
done